.github/ISSUE_TEMPLATE/Update A-enviar-software-para-revision.md

```diff
@@ -0,0 +1,105 @@
+	- [ ] análisis de texto (text analysis)
+	
+- Explica cómo y por qué el paquete cae dentro de estas categorías (1 a 3 oraciones):
+
+
+- ¿Cuál es la audiencia esperada y las aplicaciones científicas de este paquete? 
+
+- ¿Hay otros paquetes de R que logren el mismo objetivo? Si los hay, ¿cómo se diferencian del tuyo, o alcanzan nuestro [criterio del mejor de su categoría (documento en Inglés)](https://ropensci.github.io/dev_guide/policies.html#overlap)?
+
+-   (Si aplica) ¿Tu paquete cumple con nuestras [guías de _Ética, Privacidad de Datos e Investigación de Sujetos Humanos_ (documento en Inglés)](https://devguide.ropensci.org/policies.html#ethics-data-privacy-and-human-subjects-research)?
+
+- Si ya has hecho una consulta de pre-envío, por favor pega el enlace al issue correspondiente, el post del foro, u otra discusión. Alternativamente, taguea al editor (con @tag) con el que te contactaste.
+
+
+
+## Revisiones Técnicas
+
+Confirma cada uno de los siguientes al tildar en la caja.  
+
+- [ ] He leído la [guía para autores (documento en Inglés)](https://devguide.ropensci.org/guide-for-authors.html) y [la guía de paquetes de rOpenSci (documento en Inglés)](https://devguide.ropensci.org/building.html).
+
+Este paquete:
+
+- [ ] no viola los Términos de Servicio de ningún servicio con los que interactúa. 
+- [ ] tiene una licencia aceptada por CRAN y OSI.
+- [ ] contiene un [archivo README con instrucciones para isntalar la versión de desarrollo](https://ropensci.github.io/dev_guide/building.html#readme). 
+- [ ] incluye [documentación con ejemplos para todas las funciones, creada con roxygen2](https://ropensci.github.io/dev_guide/building.html#documentation).
+- [ ] contiene una viñeta con ejemplos de sus funciones esenciales y su uso.
+- [ ] tiene una [suite de tests (documento en Inglés)](https://ropensci.github.io/dev_guide/building.html#testing).
+- [ ] tiene [integración contínua (documento en Inglés)](https://ropensci.github.io/dev_guide/ci.html), incluyendo reporte de cobertura de tests, usando servicios como Travis CI, Coveralls y/o CodeCov.
+
+## Opciones de Publicación
+
+- [ ] ¿Tienes intenciones de subir este paquete a CRAN?  
+- [ ] ¿Tienes intenciones de enviar este paquete a Bioconductor?  
+- [ ] ¿Deseas automáticamente enviar a [Journal of Open Source Software (sólo en Inglés)](http://joss.theoj.org/)? En ese caso:
+
+<details>
+ <summary>Opciones para JOSS</summary>  
+
+  - [ ] Este paquete tiene una **obvia aplicación para investigación**, de acuerdo con la [definición de JOSS (documento en Inglés)](https://joss.readthedocs.io/en/latest/submitting.html#submission-requirements).
+    - [ ] Este paquete contiene un `paper.md` siguiendo los  [requerimientos de JOSS (documento en Inglés)](https://joss.readthedocs.io/en/latest/submitting.html#what-should-my-paper-contain), con una descripción detallada en la raíz del paquete o en `inst/`.
+    - [ ] El paquete está depositorado en un repositorio a largo plazo con el DOI: 
```

Comment:
Should be "depositado" -- not "deposit[or]ado".

Also, is the final colon ":" intentional? Or should it be a period "." instead? (It's strange that following this colon I see another item at the same level of indentation).